rundeckapp/grails-app/views/user/login.gsp

```diff
@@ -190,6 +191,16 @@
 
       </div>
     </div>
+      <script type="text/javascript">
+          function onLoginClicked() {
+            let lbtn = jQuery("#btn-login")
+            if(jQuery('#login').val() === '') return false
+            let spinner = jQuery("#login-spinner")
+            lbtn.hide()
```

Comment:
Wouln't be easier to just disable the button instead of hiding it and placing another element on top?
jQuery("#btn-login").prop( "disabled", true );